source/platforms/databricks.md

```diff
@@ -42,7 +42,7 @@ To get started, navigate to the **All Purpose Compute** tab of the **Compute** s
 
 ![Screenshot of the Databricks compute page](../images/databricks-create-compute.png)
 
-In order to launch a GPU node uncheck **Use Photon Acceleration** and select `15.4 LTS ML (includes Apache Spark 3.5.0, GPU, Scala 2.12)` runtime version.
+In order to launch a GPU node uncheck **Use Photon Acceleration** and select any `15.x` or `16.x` ML runtime with GPU support. For example for long-term support releases you could select the `15.4 LTS ML (includes Apache Spark 3.5.0, GPU, Scala 2.12)` runtime version.
```

Comment:
```suggestion
In order to launch a GPU node uncheck **Use Photon Acceleration** and select any `15.x` or `16.x` ML runtime with GPU support.
For example for long-term support releases you could select the `15.4 LTS ML (includes Apache Spark 3.5.0, GPU, Scala 2.12)` runtime version.
```

I have a slight preference for breaking up markdown into 1-sentence-per-line, for easier view in `git` diffs and more-granular history in the git blame. The line break won't change the visual appearance of the rendered markdown at all.

Totally a personal-preference thing, do whatever you want with that suggestion.